server/src/main/java/io/deephaven/server/console/ConsoleServiceGrpcImpl.java

```diff
@@ -276,7 +276,13 @@ public StreamObserver<AutoCompleteRequest> autoCompleteStream(
                             "from deephaven_internal.auto_completer import jedi_settings ; jedi_settings.set_scope(globals())");
                     settings[0] = (PyObject) scriptSession.getVariable("jedi_settings");
                 } catch (Exception err) {
-                    log.error().append("Error trying to enable jedi autocomplete").append(err).endl();
+                    if (err.getMessage().contains("No module named 'jedi'")) {
```

Comment:
Can we also have a flag set so that this only logs once, instead of once per stream? As it is, if you wait until the stream timeout (30s by default, this is specific for autocomplete and not a grpc thing) a new stream will start from the same user and you'll get another log message.

That said, we might want something stronger than INFO, since that won't show up in the web console, only in the server's log or in the web Log tab.